README.md

```diff
@@ -8,7 +8,7 @@ methods.
 component, or package level; has no public methods for configuration; and
 encourages typehinted retrieval methods.
 
-This means that you `composer require capsule/dev` into your package, extend
+This means that you `composer require capsule/di` into your package, extend
```

Comment:
There is a question here. 

Do you think we need a `can` ? `This means that you can`